tests/pflang-reg/pl-bug132-icmp6_or_portrange

```diff
@@ -0,0 +1 @@
+../../env pipe-lua-libpcap-match ../data/wingolog.pcap "icmp6 or portrange 14682-50101" 14951
```

Comment:
I actually hacked that in 15 minutes ago, then took it back out; shall do. I don't really like how it's making a one-line script into a 3-line one, with the first two lines identically every time, but that's not utterly terrible.